### DISCUSSION TITLE
Joomla automatic update

### DISCUSSION THREAD

**Discussion by komir2004** [akeeba/panopticon#514] [Q&A (Community Support)] [answered]
Hello, I am not sure why you have Joomla update notifications on all pages.
"The update is not scheduled to run automatically. You can run the update manually on your site, or click on the button below."
The previous automatic update was functioning properly.

**Comment by nikosdion** [comment]
I have literally no idea whatsoever what the heck you are talking about.

Remember, when trying to discuss something with other people, we are neither reading your mind, nor sitting behind your shoulder.

**Comment by komir2004** [comment]
My mistake.
The problem is this.
After a new Joomla update (5.0.1 to 5.0.2 and 4.4.1 to 4.4.2), in Panopticon I get the message "The update is not scheduled to run automatically. You can run the update manually on your site, or click on the button below."
![Akeeba-Panopticon](https://github.com/akeeba/panopticon/assets/17553018/6edf61cc-d2ab-45da-8a3e-3622e710300b)

The previous Joomla automatic update was functioning properly.
Is it possible that I disable automatic updates somewhere?
Inside Configuration, When Should Updates Be Installed has value "Immediately" 

![Akeeba-Panopticon333](https://github.com/akeeba/panopticon/assets/17553018/27a18d2b-947e-4afe-b02e-75752168c3a0)

**Comment by komir2004** [comment]
Now I notice there is a notification: "ACTION REQUIRED. The Joomla!™ update task is stuck."
![Akeeba-Panopticon444](https://github.com/akeeba/panopticon/assets/17553018/c518f72b-d846-409d-9a1b-e649cec75b2a)

Where can I see where the update was stuck?
Inside Joomla, I don't see any notifications.
The backup was executed without any problems
![Akeeba-Backup-Professional-for-Joomla-trade-Adr-Administration](https://github.com/akeeba/panopticon/assets/17553018/a8a6ef38-5ede-4bdc-a5f1-56d6d3555804)

**Comment by nikosdion** [comment, marked as answer]
This is a bug in the currently published version. It is already fixed for 1.1.0.

Kindly note that due to other things going on in meatspace right now for me I did not have the time to make a release this past week as was my original plan. I hope that next week I will have the time. I can tell you that Monday is the absolutely least likely day to do that, due to prior engagements.
